Makefile

```diff
@@ -4,13 +4,14 @@ TAG ?= $(shell git describe --tag --always --dirty)
 REGISTRY_AND_USERNAME := $(REGISTRY)/$(USERNAME)
 RUN_TESTS ?= TestIntegration
 
-PLATFORM ?= linux/amd64
+BUILD_PLATFORM ?= linux/amd64
+PLATFORMS ?= linux/amd64,linux/arm64
```

Comment:
super nit: What I tried to do in the template is to have `Makfile` vars match the `buildx` arg. Also other projects have `PLATFORM`. I can live with this but just wanted to point out the reasoning.

Reply:
makes sense, I will adjust that back to `PLATFORM`!